demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -42,17 +77,13 @@ data:
             # we want to increment this counter by 1 for each unique (source, target, service, method, response_code) tuple
             value: "1"
             labels:
-              source: source.service | "unknown"
+              source: source.name | "unknown"
```

Comment:
There should be source.name, target.name, and target.service.

Think of name as borg job, service as blade name. Only targets have blade names, but both sources and targets have job names.